### ISSUE TITLE
using travel function errors out with true_divide error

### ISSUE THREAD

**Issue by KKnoernschild** [fsc-mib/travel#2]
I am attempting run a single nii.gz lung image as input using the travel function.

running the function starts off fine, but I get a true_divide error in an skimage function. looking at the output lung volume mask that has been generated, it is just a vertical stripe that is outputted outside of the region that the lung ct covers. This is what is causing that divide error.

Is there any idea how to fix this?

I am currently attempting the travel_nora function currently, and it is still running. No immediate erroring out like the travel function.

**Comment by fsc-mib** [comment]
It seems that there is a problem with the created lung mask. We have used the following package: https://github.com/healthcAIr/LuVoX

However, you can use any other package to create the lung mask. Internally, we run an AI-based algorithm and provide its results as lung mask in the travel_nora function.

 

 

Von: KKnoernschild ***@***.***> 
Gesendet: Donnerstag, 18. August 2022 21:12
An: fsc-mib/travel ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [fsc-mib/travel] using travel function errors out with true_divide error (Issue #2)

 

I am attempting run a single nii.gz lung image as input using the travel function.

running the function starts off fine, but I get a true_divide error in an skimage function. looking at the output lung volume mask that has been generated, it is just a vertical stripe that is outputted outside of the region that the lung ct covers.

Is there any idea how to fix this?

I am currently attempting the travel_nora function currently, and it is still running. No immediate erroring out like the travel function.

**Comment by KKnoernschild** [comment]
Thank you for the help, Ill take a look at that git repo.

I did end up being able to get the travel_nora function to work using a very rough input lung mask that I created from thresholding my test image and doing some iterative filling. I poured over the travel function code for a while last night and just couldnt seem to find a reason for the lung mask failing. maybe in the future I can look more and see why that happens.

My results look promising using that rough masking method with travel_nora, so with a better overall lung segmentation I have high hopes.
